x-pack/plugins/infra/public/components/logging/log_analysis_setup/missing_privileges_messages.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const missingMlResultsPrivilegesTitle = i18n.translate(
```

Comment:
Would there be any harm in rolling `missingMlResultsPrivilegesTitle` and `missingMlSetupPrivilegesTitle` into a singular translation as they use the same text?

Reply:
good point, I didn't realize that when I factored them out :+1: